src/Razor/src/Microsoft.CodeAnalysis.Razor/RefTagHelperDescriptorProvider.cs

```diff
@@ -61,16 +61,24 @@ private TagHelperDescriptor CreateRefTagHelper()
                 });
             });
 
-            builder.BindAttribute(attribute =>
+            builder.BindAttribute(@ref =>
             {
-                attribute.Documentation = ComponentResources.RefTagHelper_Documentation;
-                attribute.Name = "@ref";
+                @ref.Documentation = ComponentResources.RefTagHelper_Documentation;
+                @ref.Name = "@ref";
```

Comment:
Hmm, At some point it'd probably make sense to have a collection of defined strings that mean something to us so we can easily see where they're used/hardcoded. Looking at `"suppressField"` below and up in the ref lowering pass made me think of this. No action, just food for thought.

Reply:
yeah I don't really like how many places these strings appear either. If you're interested in this do you want to send a PR with a proposal.